### PR TITLE
fix(ui): Change UI container port to 8080 to work with new nginx image

### LINKED ISSUE #72: Syndesis-ui probes failing since switch to nginx
Related to https://github.com/syndesisio/syndesis-ui/issues/622

Any suggestion for the new port to check?

```
Readiness probe failed: Get http://10.1.15.49:2015/: dial tcp 10.1.15.49:2015: getsockopt: connection refused
```

### PR DESCRIPTION
**Pull request by jimmidyson**
Fixes #72

### CODE REVIEW

**Review by rhuss — CHANGES_REQUESTED** [review]
You should regenerate the templates, otherwise lgtm

**Comment by rhuss** [comment]
@jimmidyson any change you can recreate the templates ? If not, no problem I can do it. But need your agreement ;-)

**Comment by jimmidyson** [comment]
Doh! Sorry - done now.

**Review by rhuss — APPROVED** [review]
LGTM

File: generator/03-syndesis-ui.yml.mustache [hunk 1]
```diff
@@ -84,12 +84,12 @@
...
+              port: 8080
             initialDelaySeconds: 3
           readinessProbe:
             httpGet:
               path: "/"
-              port: 2015
+              port: 8080
             initialDelaySeconds: 1{{/Probeless}}
           ports:
           - containerPort: 2015
```
Comment (rhuss) [inline generator/03-syndesis-ui.yml.mustache:95]: @jimmidyson shouldn't this be adapted to 8080, too ?

**Comment by jimmidyson** [comment]
Bugger - submitted #74

**Pull request merged**